build.gradle

```diff
@@ -18,32 +18,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
-/**
-* The application plugin facilitates creating an executable JVM application.
-*/
+ 
+ /**
+ * The application plugin facilitates creating an executable JVM application.
+ */
 
 apply plugin: 'application'
-apply plugin: 'eclipse'
-
-/**
-* This line declares a Gradle project nature for the Eclipse IDE.
-*/
 
-eclipse.project {
-  	natures 'org.springsource.ide.eclipse.gradle.core.nature'
-}
-
-/**
-* This line specifies where the main.java file is found.
-*/
+ /**
+ * This line specifies where the main.java file is found.
+ */
 
 mainClassName='View.Main'
 
-/**
-* This statement declares where Gradle is to find directories
-* for source code, external resources, and test classes
-*/
+ /**
```

Comment:
This slash in the first line of a multi-line comment does not get indented with a space.

Reply:
Finished & pushed